src/main/java/net/sf/jabref/model/ChainNode.java

```diff
@@ -126,6 +126,7 @@ protected void setParent(T parent) {
      *
      */
     public void removeChild() {
+        // NPE if this is ever called
         child.setParent(null);
```

Comment:
Isn't it better that the author of the faulty code fixes it when the implementation is finished? I do not really see how this code was ever approved...